apps/condo/domains/ticket/components/TicketForm/CreateTicketForm.tsx

```diff
@@ -50,24 +50,30 @@ export const CreateTicketForm: React.FC = () => {
     const router = useRouter()
     const auth = useAuth() as { user: { id: string } }
 
+    const organizationRef = useRef()
```

Comment:
it looks a little bit hacky

Reply:
Maybe it's better to override the `useOrganization` context for this component?

Reply:
remove it and use `useCallback` depends on `organization`